composer.json

```diff
@@ -1,32 +1,41 @@
 {
-    "name": "yiisoft/_____",
+    "name": "yiisoft/auth",
     "type": "library",
-    "description": "_____",
-    "keywords": ["_____"],
+    "description": "Yii auth",
+    "keywords": ["auth", "middleware"],
     "homepage": "http://www.yiiframework.com/",
     "license": "BSD-3-Clause",
     "support": {
-        "issues": "https://github.com/yiisoft/_____/issues?state=open",
+        "issues": "https://github.com/yiisoft/auth/issues?state=open",
         "forum": "http://www.yiiframework.com/forum/",
         "wiki": "http://www.yiiframework.com/wiki/",
         "irc": "irc://irc.freenode.net/yii",
-        "source": "https://github.com/yiisoft/_____"
+        "source": "https://github.com/yiisoft/auth"
     },
+    "minimum-stability": "dev",
     "require": {
-        "php": "^7.2"
+        "php": "^7.2",
+        "psr/container": "^1.0",
+        "psr/http-server-middleware": "^1.0",
+        "psr/http-message": "^1.0",
+        "psr/http-server-handler": "^1.0",
+        "yiisoft/strings": "^3.0@dev",
+        "yiisoft/yii-web": "^3.0@dev"
```

Comment:
That creates circular dependency: `yii-web` depends on `auth` and `auth` depends on `yii-web`. I expect trouble with circular breaks that hard to fix.

We have the following used from `yii-web` in this package:

```
use Yiisoft\Yii\Web\User\IdentityInterface;
use Yiisoft\Yii\Web\User\IdentityRepositoryInterface;
```

How about moving these to this package? These are about auth anyway.

Reply:
> How about moving these to this package? These are about auth anyway.

:thinking: I'm thinking about splitting `user` package :)

Reply:
That would cause the same problem and you'll have to split:

1. user
2. session
3. cookie

That's a bit too much, IMO.

Reply:
OK. `AuthenticationKeyInterface` as well?

Reply:
No. That is "remember me" and isn't directly connected to authentication itself.